code/modules/reqs/supply.dm

```diff
@@ -648,6 +649,11 @@ GLOBAL_LIST_INIT(blacklisted_cargo_types, typecacheof(list(
 		supply_interface = new(src)
 	return supply_interface.interact(user)
 
+/obj/item/storage/backpack/marine/radiopack/process()
+	var/datum/component/beacon/b = GetComponent(/datum/component/beacon)
```

Comment:
> I think this is the intended way the radiopack was supposed to work in the first place, instead of it acting as a beacon on the place where it was innitially activated.

Proof? Also this is a bad way of doing this. Never ever ever ever do this, this is not how components are meant to be used.

Reply:
Zero proof, hence me prefacing it with "I think". 
How are components meant to be used?

Reply:
https://hackmd.io/@tgstation/SignalsComponentsElements